src/features/exportStatic.ts

```diff
@@ -50,4 +47,25 @@ export default (api: IApi) => {
       return memo;
     },
   });
+
+  api.onGenerateFiles(() => {
+    api.writeTmpFile({
+      path: 'core/exportStaticRuntimePlugin.ts',
```

Comment:
dumi 的插件统一写到 dumi 下哈
```suggestion
      path: 'dumi/exportStaticRuntimePlugin.ts',
```

Reply:
done